WordPress/src/main/java/org/wordpress/android/ui/accounts/login/LoginEmailFragment.java

```diff
@@ -97,6 +97,7 @@ protected void setupContent(ViewGroup rootView) {
             @SuppressWarnings("PrivateMemberAccessBetweenOuterAndInnerClass")
             @Override
             public void onClick(View view) {
+                AnalyticsTracker.track(AnalyticsTracker.Stat.LOGIN_SOCIAL_BUTTON_CLICK);
                 AnalyticsTracker.track(AnalyticsTracker.Stat.LOGIN_GOOGLE_TAPPED);
```

Comment:
This seems like another redundant stat.  Should we remove `LOGIN_GOOGLE_TAPPED` in favor of `LOGIN_SOCIAL_BUTTON_CLICK`?

Reply:
Yeah. I'll remove this one too.